### PR TITLE
fix(DB/Quest): Return To Obadei

### PR DESCRIPTION
**Pull request by Vanheden**
## Changes Proposed:

Fixes the dialogue .

## Issues Addressed:

Fixed the NPC Text after Quest Completion.

Closes: https://github.com/azerothcore/azerothcore-wotlk/issues/16199

## SOURCE:

Retail WoW & Youtube Videos.

## Tests Performed:

Tried it out ingame and it works and here's a video of it.

https://www.youtube.com/watch?v=plEzy8o2eu4&ab_channel=Roddan

## How to Test the Changes:

1. .q add 9423
2. .q complete 9423
3. Turn in the Quest.

## Known Issues and TODO List:

None that I'm aware of.

### CODE REVIEW

File: data/sql/updates/pending_db_world/ReturnToObadei.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+(16833, 0, 0, 'No!  Not... Sedai!  The orcs must pay!', 12, 0, 100, 0, 0, 0, 13997, 0, 'Makuru // Makuru');
+
+ -- Anchorite Obadei smart ai
+SET @ENTRY := 16834;
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryOrGuid` = @ENTRY;
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` = @ENTRY;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(@ENTRY, 0, 0, 0, 20, 0, 100, 0, 9423, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 10, 57906, 16833, 0, 0, 0, 0, 0, 'On player rewarded quest Return to Obadei (9423) - Creature Makuru (16833) with guid 57906 (fetching): Talk No!  Not... Sedai!  The orcs must pay! (0) to invoker');
+
+
```
Comment (Maria-sequel) [inline data/sql/updates/pending_db_world/ReturnToObadei.sql:13]: extra whitespaces here and at the bottom

**Review by Maria-sequel — CHANGES_REQUESTED** [review]

**Comment by GastonCK** [comment]
Author provided a video testing it, does this need another test?

**Comment by Nyeriah** [comment]
Please fix the SQL file (remove the extra blank lines) so this can be merged

**Comment by elthehablo** [comment]
abandoned?

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**